src/app/containers/manuscript/styles.ts

```diff
@@ -3,21 +3,20 @@ import { makeStyles } from '@material-ui/core/styles';
 const tocWidth = 240;
 
 export const useManuscriptStyles = makeStyles((theme) => ({
+  contentWrapper: {
+    [theme.breakpoints.up('sm')]: {
+      paddingLeft: tocWidth + theme.spacing(2)
+    }
+  },
   content: {
-    maxWidth: 700 + tocWidth,
```

Comment:
The keywords components are now missing a margin between them, which i think is a side effect of the below. You can easily see the issue by comparing the staging version with a local version with these changes.

Reply:
not caused by this but the problem exists. Fixed it now